vector/src/main/java/im/vector/riotx/features/home/room/detail/timeline/helper/MessageInformationDataFactory.kt

```diff
@@ -72,6 +75,52 @@ class MessageInformationDataFactory @Inject constructor(private val session: Ses
             textColor = colorProvider.getColor(getColorFromUserId(event.root.senderId))
         }
 
+        val room = event.root.roomId?.let { session.getRoom(it) }
+        val e2eDecoration: E2EDecoration
```

Comment:
merge with line 81:
val e2eDecoration = if()... and remove all the affectation in blocks, for shorter code

Reply:
done